contracts/bid/PartyBid.sol

```diff
@@ -0,0 +1,235 @@
+/*
+
+      ___           ___           ___           ___           ___           ___                       ___
+     /\  \         /\  \         /\  \         /\  \         |\__\         /\  \          ___        /\  \
+    /::\  \       /::\  \       /::\  \        \:\  \        |:|  |       /::\  \        /\  \      /::\  \
+   /:/\:\  \     /:/\:\  \     /:/\:\  \        \:\  \       |:|  |      /:/\:\  \       \:\  \    /:/\:\  \
+  /::\~\:\  \   /::\~\:\  \   /::\~\:\  \       /::\  \      |:|__|__   /::\~\:\__\      /::\__\  /:/  \:\__\
+ /:/\:\ \:\__\ /:/\:\ \:\__\ /:/\:\ \:\__\     /:/\:\__\     /::::\__\ /:/\:\ \:|__|  __/:/\/__/ /:/__/ \:|__|
+ \/__\:\/:/  / \/__\:\/:/  / \/_|::\/:/  /    /:/  \/__/    /:/~~/~    \:\~\:\/:/  / /\/:/  /    \:\  \ /:/  /
+      \::/  /       \::/  /     |:|::/  /    /:/  /        /:/  /       \:\ \::/  /  \::/__/      \:\  /:/  /
+       \/__/        /:/  /      |:|\/__/     \/__/         \/__/         \:\/:/  /    \:\__\       \:\/:/  /
+                   /:/  /       |:|  |                                    \::/__/      \/__/        \::/__/
+                   \/__/         \|__|                                     ~~                        ~~
+
+PartyBid v3
```

Comment:
can we call this either v2 or something else

Reply:
I believe we called PartyBid with splits v2 - we added a number variable that set VERSION = 2 for that one - so if we are going by versions this one would be 3

do you have an idea for what else you would call it?